packages/pds/service/index.js

```diff
@@ -36,7 +36,10 @@ const main = async () => {
   const cfInvalidator = new CloudfrontInvalidator({
     distributionId: env.cfDistributionId,
   })
-  const signingKeypair = await KmsKeypair.load({
+  const repoSigningKey = await KmsKeypair.load({
+    keyId: env.signingKeyId,
+  })
+  const plcRotationKey = await KmsKeypair.load({
     keyId: env.signingKeyId,
   })
```

Comment:
I think it's right, just confirming both are intended to be `env.signingKeyId` for good measure.

Reply:
Was able to clear this up myself by rereading the message you sent 😎